nemo_skills/inference/litellm_hybrid_cache.py

```
@@ -0,0 +1,240 @@
+            self._dirty = True
+
+    async def async_set_cache(self, key, value, **kwargs):
+        """Async set - delegates to sync implementation since we're using in-memory."""
+        self.set_cache(key=key, value=value, **kwargs)
+
+    async def async_set_cache_pipeline(self, cache_list, **kwargs):
+        """Set multiple cache entries."""
+        for cache_key, cache_value in cache_list:
+            self.set_cache(key=cache_key, value=cache_value, **kwargs)
+
+    def get_cache(self, key, **kwargs):
```

Ruff (0.14.14)

[warning] Unused method argument: `kwargs`

(ARG002)